pkg/scaffold/olm-catalog/config.go

```diff
@@ -0,0 +1,130 @@
+// Copyright 2018 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package catalog
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/operator-framework/operator-sdk/pkg/scaffold"
+
+	"github.com/ghodss/yaml"
+)
+
+// CSVConfig is a configuration file for CSV composition. Its fields contain
+// file path information.
+type CSVConfig struct {
+	OperatorPath string   `json:"operator-path,omitempty"`
+	CRDCRPaths   []string `json:"crd-cr-paths,omitempty"`
+	RolePath     string   `json:"role-path,omitempty"`
```

Comment:
We should add some comments similar to what we have in the design doc on what these fields are and what they default to.
https://github.com/operator-framework/operator-sdk/blob/master/doc/design/milestone-0.2.0/csv-generation.md#configuration
It's not immediately obvious what OperatorPath is if you look at it for the first time.